packages/bot-web-ui/src/components/dashboard/dashboard-component/cards.tsx

```diff
@@ -10,14 +10,15 @@ import Recent from './load-bot-preview/recent';
 import GoogleDrive from './load-bot-preview/google-drive';
 
 type TCardProps = {
-    closeResetDialog: VoidFunction;
     dialog_options: { [key: string]: string };
-    handleFileChange: (e: React.ChangeEvent, flag?: boolean) => boolean;
     has_dashboard_strategies: boolean;
     is_dialog_open: boolean;
     is_mobile: boolean;
-    loadFileFromLocal: VoidFunction;
     save_modal: SaveModalStore;
+    closeResetDialog: VoidFunction;
+    handleFileChange: (e: React.ChangeEvent, flag?: boolean) => boolean;
+    loadFileFromLocal: VoidFunction;
+    loadDataStrategy: VoidFunction;
```

Comment:
We are using type VoidFunction here. @maryia-matskevich-deriv

Reply:
@sandeep-deriv done